R/Filtering.R

```diff
@@ -42,7 +42,7 @@ FilterFields <- function(.data) {
     "ActivityMediaSubdivisionName",
     "ActivityCommentText", "MonitoringLocationTypeName",
     "StateName", "TribalLandName",
-    "OrganizationFormalName", "CharacteristicName",
+    "OrganizationFormalName", "TADA.CharacteristicName",
```

Comment:
Do the changes made to autoclean impact other recommended filter fields here (e.g. media?)

Reply:
This is a good question. There are some instances in Filtering.R where I converted CharacteristicName to TADA.CharacteristicName, but retained ActivityMediaName even though there's now a TADA.ActivityMediaName. This was probably an oversight on my part. But it brings up a good question of whether these filtering functions should focus on the original columns or the TADA columns, where applicable.

Reply:
In this case with media, the function should use TADA.ActivityMediaName, because that is where we already address issues of upper/lowercase (e.g. Water and water, are both now WATER in TADA.ActivityMediaName). We should not create TADA versions of the columns unless there is a reason to do so, and from that perspective, my inclination is that if there is a TADA field, it should be used for the filtering (and other subsequent functions in the workflow). Previously this was not an issue since we didn't have multiple copies of that same fields that only differ in case.

Reply:
This logic makes sense to me.